src/energyplus/ForwardTranslator/ForwardTranslateSizingPlant.cpp

```diff
@@ -68,6 +68,9 @@ namespace energyplus {
       case openstudio::IddObjectType::OS_DistrictHeating: {
         return PlantSizingType::HOTWATER;
       }
+      case openstudio::IddObjectType::OS_Chiller_Electric_ASHRAE205: {
+        return PlantSizingType::CHILLEDWATER;
+      }
```

Comment:
No shenanigans needed, it'll always be on the supply side of only one Plant Loop.